__tests__/app.test.js

```diff
@@ -108,5 +108,21 @@ describe("app.js", () => {
       });
       expect(comments).toBeSorted({ key: "created_at", descending: true });
     });
+    test("GET 404: responds to the client with an error message when the article is is valid but no matching comments are found", async () => {
+      const response = await request(app)
+        .get("/api/articles/1000/comments")
+        .expect(404);
+      const { body } = response;
+      const { message } = body;
+      expect(message).toBe("No comments matching the provided article ID");
```

Comment:
This is a good test, but it's more specifically a case of _no article found on the given ID_